adoc/extensions/sycl_khr_group_interface.adoc

```diff
@@ -0,0 +1,479 @@
+[source,role=synopsis,id=api:khr-group-interface-common-group-rank_dynamic]
+----
+static constexpr rank_type rank_dynamic() noexcept;
+----
+
+_Effects_: Equivalent to [code]#return extents_type::rank_dynamic();#.
+
+'''
+
+.[apidef]#GroupT::static_extent#
+[source,role=synopsis,id=api:khr-group-interface-common-group-static_extent]
+----
+static constexpr size_t static_extent(rank_type r) noexcept;
+----
+
+_Effects_: Equivalent to [code]#return extents_type::static_extent(r);#.
+
+'''
+
+.[apidef]#GroupT::size#
+[source,role=synopsis,id=api:common-group-size]
+----
+constexpr size_type size() const noexcept;
+----
+
+_Returns_: The total number of work-items in the group, equal to the product of
+the number of work-items in each dimension of the group.
+
+'''
+
+[[sec:khr-group-interface-common-non-member-funcs]]
+==== Non-member functions
+
+.[apidef]#get_item#
+[source,role=synopsis,id=api:common-group-get-item]
+----
+template <typename Group>
+work_item<Group> get_item(Group g) noexcept;
+----
+
+_Constraints_: Available only if `Group` is `work_group` or `sub_group`.
+
+_Returns_: A [code]#work_item# representing the calling work-item within group
+[code]#g#.
+
+'''
+
+.[apidef]#leader_of#
+[source,role=synopsis,id=api:common-group-leader_of]
+----
+template <typename Group>
+bool leader_of(Group g) noexcept;
+----
+
+_Returns_: [code]#true# if the calling work-item is the leader of group
+[code]#g#, and [code]#false# otherwise.
+
+_Remarks_: [code]#leader_of# returns [code]#true# for only one work-item in a
+group.
+The leader of the group is determined during construction of the group, and is
+invariant for the lifetime of the group.
+The leader of the group is guaranteed to be the work-item with index 0 within
+the group.
+
+[[sec:khr-group-interface-work_group]]
+== [code]#work_group# class
+
+The [code]#work_group# class template encapsulates all functionality required to
+represent a particular <<work-group>> within a kernel.
+It is not user-constructible.
+
+The SYCL [code]#work_group# class template provides common by-value semantics
+(see <<sec:byval-semantics>>) and the common group interface (see
+<<sec:khr-group-interface-common>>).
+
+[source,role=synopsis]
+----
+namespace sycl {
+namespace khr {
+
+template <int Dimensions = 1>
+class work_group {
+
+ public:
+  using id_type = id<Dimensions>;
+  using linear_id_type = size_t;
+  using range_type = range<Dimensions>;
+#if defined(__cpp_lib_mdspan)
+  using extents_type = std::dextents<size_t, Dimensions>;
+#endif
+  using size_type = size_t;
+  static constexpr int dimensions = Dimensions;
+  static constexpr memory_scope fence_scope = memory_scope::work_group;
+
+  work_group(group<Dimensions> g) noexcept;
+
+  operator group<Dimensions>() const noexcept;
+
+  /* -- common by-value interface members -- */
+
+  /* -- common group interface members -- */
+
+};
+
+} // namespace khr
+} // namespace sycl
+----
+
+.[apidef]#work_group constructor#
```

Comment:
```suggestion
.[apititle]#work_group constructor#
```

Use `[apititle]` when the text is not a C++ identifier.

Reply:
Fixed as part of de356628056c3d1034fcde2e2be59f9c4ebf3ecc.